lib/models/store_transaction.dart

```diff
@@ -25,11 +27,17 @@ class StoreTransaction with _$StoreTransaction {
 
     /// Purchase date of the transaction in ISO 8601 format.
     String purchaseDate,
+  {
+    /// Deprecated: Use transactionIdentifier instead.
+    @Deprecated('Use orderId instead.')
+    /// RevenueCat Id associated to the transaction.
+    @Default('') String transactionIdentifier,
+  }
```

Comment:
This needs to be made optional in order to have a default, which it actually makes it a breaking change